lowatt_grdf/api.py

```diff
@@ -151,9 +151,18 @@ def declare_acces(self, access: models.DeclareAccess) -> None:
         )
         LOGGER.info("Successfully declared access to %s", access.pce)
 
-    def donnees_consos_publiees(self, pce: str, from_date: str, to_date: str) -> Any:
+    def donnees_consos_publiees(
+        self, pce: str, from_date: str = None, to_date: str = None, periode: str = None
```

Comment:
This should be `Optional[None]` also since kwargs are in english use `period` here.